apps/yapms/src/lib/components/modals/editregionmodal/EditRegionModal.svelte

```diff
@@ -38,15 +31,8 @@
 			required
 			bind:value={newValue}
 		/>
-		<div class="modal-action">
-			<button class="btn btn-primary" on:click={close}> No </button>
-			<button
-				class="btn btn-success"
-				on:click={confirm}
-				disabled={newValue === null || newValue < 0 || newValue % 1 !== 0}
-			>
-				Update
-			</button>
-		</div>
 	</div>
-</div>
+	<div slot="action">
+		<button class="btn btn-success" on:click={confirm}>Confirm</button>
```

Comment:
This button needs to have the conditions previously applied to it or they need to be checked in confirm. A negative number entered will brick the app. Null will trap you in the modal.